app/views/pages/privacy_policy.html.erb

```diff
@@ -0,0 +1,193 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+  <h1 class='govuk-heading-xl'>
+    Privacy Notice: Teachers: claim back your student loan repayments
+  </h1>
+
+    <h2 class='govuk-heading-l'>
+      Who we are
+    </h2>
+
+    <p class="govuk-body">
+      This work is being carried out by the Department for Education (DfE) Digital, which is a part of DfE. DfE engages the
+      private company dxw to help improve and provide the service. For the purpose of data protection legislation, DfE
+      is the data controller for the personal data processed as part of eachers: claim back your student loan repayments.
+      eachers: claim back your student loan repayments is a free and optional service eligible teachers to claim back
+      student loan repayments.
+    </p>
+
+    <h2 class='govuk-heading-l'>
+      When we collect personal information
+    </h2>
+
+    <p class="govuk-body">
+      We receive your personal data when you submit it:
+    </p>
+
+    <ul class='govuk-list govuk-list--bullet'>
+      <li>when contacting us for support using the service, to leave feedback or for any other matters</li>
+      <li>as a user of the service</li>
+    </ul>
+
+    <h2 class='govuk-heading-l'>
+      What personal information we will collect
+    </h2>
+
+    <p class="govuk-body">
+      We will collect:
+    </p>
+
+    <ul class='govuk-list govuk-list--bullet'>
+      <li>
+        your name, email and (if applicable) your phone number when you contact us
```

Comment:
This list is not complete. Do we need to mention all the personal information we capture as part of the claim process?

Reply:
Yeah I think when we have that worked out, and have down what we will get from DQT and Teacher pensions then we should resist this. Backlog story so we don't forget?